### ISSUE TITLE
Component CampingSiteExplorer.svelte's buttons should be bigger on small screens

### ISSUE THREAD

**Issue by jackson-wilmot** [jackson-wilmot/national-park-website#3]
# Description
The buttons in CampingSiteExplorer.svelte should be larger to improve UX and a11y.
